modelforge/utils/remote.py

```diff
@@ -61,7 +60,7 @@ def fetch_url_from_doi(doi: str, timeout: Optional[int] = 10) -> str:
     import requests
 
     # force to use ipv4; my ubuntu machine is timing out when it first tries ipv6
-    #requests.packages.urllib3.util.connection.HAS_IPV6 = False
+    # requests.packages.urllib3.util.connection.HAS_IPV6 = False
```

Comment:
mine too! I had to delete this test previously

Reply:
oh, that is an old comment!  I ended up fixing it, it was a weird settings error.  I'm going to leave that import in, commented out, just in case we run into this again.  This is not a very used function anymore, after changing how we structured some of the code.